crates/router/src/connector/noon.rs

```diff
@@ -58,22 +57,49 @@ impl
     // Not Implemented (R)
 }
 
+fn get_auth_header(
```

Comment:
Keep this inside ConnectorCommon itself, Since &settings::Connectors is no more required

Reply:
We would need to consume test_mode from router_data so we can't keep it inside ConnectorCommon